data/human/transport missions.txt

```diff
@@ -3582,3 +3582,297 @@ mission "Hauler VI cargo prototype 3"
+					decline
+			`	"Indeed? Superb! Come back as soon as possible. We wouldn't want to risk your friend's life for a minute delay."`
+				accept
+			label `second chance`
+			`	On the way to the spaceport you realize that you do not have to find another spaceship captain to bring medicine from <planet>. You could do it yourself!`
+			choice
+				`	(Take the job.)`
+					accept
+				`	(Find someone else.)`
+					decline
+	on accept
+		event "james medicine too slow" 20
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+			confusion 20
+		system "Alpha Hydri"
+		fleet "Small Northern Pirates"
+
+event "james medicine too slow"
+
+mission "James Medicine [1]"
+	landing
+	name "Medicine from <origin>"
+	description "James's doctors have asked you to bring some newly-developed remedy from <planet> to <origin> as quickly as possible, even though they are not really sure what his illness is."
+	cargo "Essential medicine" 1
+	deadline
+	source "Nimbus"
+	destination "Hestia"
+	to offer
+		has "James Medicine [0]: done"
+	blocked "You have reached <origin>, but you need <capacity> in order to take the medicine for James. Return here when you have the required space free."
+	on visit
+		dialog "generic cargo on visit"
+	on offer
+		dialog `You take the medicine from a newly-constructed section of the spaceport, as instructed. Now you just have to deliver it back to <planet>.`
+	on complete
+		conversation
+			`Landing again on <planet>, you hurry to the Pilot Veteran Clinic with the medicine from <origin>.`
+			branch slow
+				has "event: james medicine too slow"
+			`	The doctor that spoke to you earlier appears soon after you enter the building. "Back already? I can hardly believe the news!" he exclaims and adds with a smile, "I have no idea how you made the trip so fast, but this little box you brought might be life-saving for your friend."`
+			choice
+				`	"I did my best."`
+					goto end
+				`	"No help is too much for a friend."`
+					goto end
+			label slow
+			`	The doctor that spoke to you earlier appears soon after you enter the building. "Alas! Such is our fate, to slowly pass from this life to the next!", he exclaims, and adds "Unfortunately, the disease was more onerous than we could have anticipated. The medicine you brought would save James's life, but the damage already wrought would leave him unable to walk freely."`
+			apply
+				set "james medicine: james disabled"
+			choice
+				`	"I tried my best."`
+					goto end
+				`	"It's your fault, not delivering for the medicine earlier."`
+			`	The doctor seems amazed by your complaint, even though he does not reply to it in any way.`
+			label end
+			branch noname
+				not "james medicine: infected"
+			`	"Anyway, we have to pay for the services we receive. Here, take <payment>", he says and drops his voice to mention, "I would not recommend seeing James now, though I will surely tell him how helpful you have been, <last>."`
```

Comment:
```suggestion
			`	"Anyway, we have to pay for the services we receive. Here, take <payment>", he says and drops his voice to mention, "I would not recommend seeing James now, though I will surely tell him how helpful you have been, Captain <last>."`
```
Just referring to someone by their last name can come across as slightly rude.

Reply:
I'm not sure if Captain <last> is capitalized in other missions.